atef/enums.py

```diff
@@ -19,3 +19,11 @@ class GroupResultMode(str, enum.Enum):
     all_ = "all"
     #: At least one item must succeed.
     any_ = "any"
+
+
+class PlanDestination(str, enum.Enum):
+    """Destination for plans to be executed"""
+    # local RunEngine
+    local_ = "local"
```

Comment:
Nit: why the `_` suffix on `local_` and `qserver_`?

Reply:
This was some combination of me mistakenly thinking `local` was a python keyword and mimicking the preceding `GroupResultMode`.  They don't need the `_` you're right